### PR TITLE
Create @wmr-plugins/directory-import

### PR DESCRIPTION
**Pull request by JoviDeCroock**
- [ ] publish
- [ ] remove from wiki

### CODE REVIEW

**Review by marvinhagemeister — APPROVED** [review]

**Review by marvinhagemeister — APPROVED** [review]

File: packages/directory-plugin/README.md [hunk 1]
```diff
@@ -0,0 +1,30 @@
+# @wmr/ls-plugin
```
Comment (marvinhagemeister) [inline packages/directory-plugin/README.md:1]: This should probably be `directory-plugin`. There are a few other old references. I think we should stick with either one and be consistent in our naming 👍

**Review by marvinhagemeister — APPROVED** [review]

File: packages/directory-plugin/README.md [hunk 2]
```diff
@@ -0,0 +1,30 @@
+# @wmr/directory-plugin
+
+Allows you to import multiple files like this:
+
+```js
+import files from 'ls:./pages';
```
Comment (marvinhagemeister) [inline packages/directory-plugin/README.md:6]: `ls:` -> `directory:`

File: packages/directory-plugin/src/index.js [hunk 3]
```diff
@@ -0,0 +1,40 @@
...
+      const resolved = await this.resolve(id.slice(10) + '\0', importer, { skipSelf: true });
+
+      if (resolved) {
+				return '\0directory:' + pathToPosix(resolved.id).replace(/\0$/, '');
+			}
+    },
+    async load(id) {
+      if (!id.startsWith('\0directory:')) return;
+
+      // remove the "\0ls:" prefix and convert to an absolute path:
```
Comment (marvinhagemeister) [inline packages/directory-plugin/src/index.js:26]: `ls:` -> `directory:`

File: packages/directory-plugin/README.md (hunk 2 above)
Comment (developit) [reply packages/directory-plugin/README.md:6]: a thought: what about `dir:`?

Comment (marvinhagemeister) [reply packages/directory-plugin/README.md:6]: I'm good with `dir` too, although the more verbose variant might be easier to for newcomers. Don't mind going either way though.

File: packages/directory-plugin/README.md [hunk 4]
```diff
@@ -0,0 +1,30 @@
...
+## or
+npm i --save @wmr-plugins/directory-import
+```
+
+## Usage
+
+Enable the plugin by importing _(or pasting!)_ it into your `wmr.config.js`:
+
+```js
+import lsPlugin from '@wmr-plugins/directory-import';
```
Comment (marvinhagemeister) [inline packages/directory-plugin/README.md:23]: Nit: Old reference to `lsPlugin`

File: packages/directory-plugin/README.md [hunk 5]
```diff
@@ -0,0 +1,30 @@
...
+
+export default function (config) {
+	config.plugins.push(lsPlugin(config));
```
Comment (marvinhagemeister) [inline packages/directory-plugin/README.md:26]: same here

File: packages/directory-plugin/example/public/index.js [hunk 6]
```diff
@@ -0,0 +1,38 @@
...
+
+import pages from 'dir:./pages';
+
+// Generate a Route component and URL for each "page" module:
+const routes = pages.map(name => ({
+  Route: lazy(() => import(`./pages/${name}`)),
+  url: '/' + name.replace(/(index)?\.\w+$/, '')  // strip file extension and "index"
+}));
+
+console.log({ pages, routes });
```
Comment (marvinhagemeister) [inline packages/directory-plugin/example/public/index.js:13]: The `console.log` is probably committed accidentally

**Review by marvinhagemeister — APPROVED** [review]

**Pull request merged**